src/product/ProductAPI.ts

```diff
@@ -377,7 +377,10 @@ export class ProductAPI {
    */
   async getBestAsksAndBids(productIds: string[]): Promise<PriceBook[]> {
     const resource = `/brokerage/best_bid_ask`;
-    const response = await this.apiClient.get(resource, {params: {product_ids: productIds}});
+    const response = await this.apiClient.get(resource, {
+      params: {product_ids: productIds},
+      paramsSerializer: serializeParamsArray,
```

Comment:
could we possibly move this [here](https://github.com/JoshJancula/coinbase-advanced-node/blob/a81429ada2ae35122911b43d451691cd155d62bd/src/client/RESTClient.ts#L83) inside the axios create so it applies to all requests?